docs/Build.md

```diff
@@ -0,0 +1,70 @@
+# Build Documentation
+
+## Build Information
+
+The build process is performed using the `build.ps1` PowerShell script located in the root of the project.
+The build script combines the module code with the generated MAML documentation.
+The only prerequisites for building this project are PowerShell and the [PlatyPS][PlatyPS] module.
+
+The build process will create the output in the `bin` directory beneath the project root.
+
+[PlatyPS]: https://github.com/PowerShell/platyPS
+
+## Building the Module
+
+To build the project from PowerShell:
+
+```powershell
+Install-Module -Scope CurrentUser PlatyPS
+git clone https://github.com/PowerShell/WindowsPowerShellCompatibilityPack.git
+Set-Location WindowsPowerShellCompatibilityPack
+./build.ps1
+```
+
+## Importing The Built Module
+
+From the project root, run the following:
+
+```powershell
+$OutputPath = Join-Path $Pwd 'bin'
+$env:PSModulePath = '{0}{1}{2}' -f $OutputPath, ([System.IO.Path]::PathSeparator), $env:PSModulePath
```

Comment:
For developers using the module out of the repo, seems like it would be easier to just import directly out of the bin folder than to modify the psmodulepath

Reply:
I've been trying to establish this pattern as a best practice. There are some subtle problems with developing a module that do not appear when you directly import a `psd1`, But I can change it if you want.

Reply:
I vaguely recall there was an issue where you imported a module via path that wasn't in PSModulePath, but don't remember what the problem was or if we fixed it in PSCore6.  My preference is if there is an issue making it harder for developers to author modules and test them, we should fix that.

Reply:
fixed.